.github/workflows/release.yml

```diff
@@ -86,13 +84,13 @@ jobs:
           default: true
           target: ${{ matrix.config.target }}
 
-      # - name: setup for cross-compiled linux aarch64 build
-      #   if: matrix.config.target == 'aarch64-unknown-linux-gnu'
-      #   run: |
-      #     sudo apt update
-      #     sudo apt install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
-      #     echo '[target.aarch64-unknown-linux-gnu]' >> ${HOME}/.cargo/config.toml
-      #     echo 'linker = "aarch64-linux-gnu-gcc"' >> ${HOME}/.cargo/config.toml
+      - name: setup for cross-compiled linux aarch64 build
+        if: matrix.config.target == 'aarch64-unknown-linux-gnu'
+        run: |
+          sudo apt update
+          sudo apt install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu
+          echo '[target.aarch64-unknown-linux-gnu]' >> ${HOME}/.cargo/config.toml
+          echo 'linker = "aarch64-linux-gnu-gcc"' >> ${HOME}/.cargo/config.toml
```

Comment:
It doesn't _seem_ like this should be necessary with `--target` set, but I don't know for sure.

Reply:
I removed these two lines and started seeing what look like similar errors from #293. Here is the latest [workflow run](https://github.com/jpflueger/spin/actions/runs/3597379547/jobs/6059072395).

Reply:
It looks like this is still necessary. In the failed workflow run the logs seem to indicate that is is still finding part of the x86_64 toolchain. Could we remove this echo by putting this configuration in the repository's `.cargo/config`? Not sure if that's a good idea or not, just thinking out loud.

Reply:
Let's just leave what we know works for now